hazelcast/src/test/java/com/hazelcast/map/query/QueryIndexingTest.java

```diff
@@ -134,41 +122,10 @@ private static Config newConfig(final Map<Integer, Employee> employees) {
             } else {
                 val = new Employee(i, "name" + i, "city" + i, 0, true, i);
             }
-            employees.put(i, Mockito.spy(val));
+            Employee spy = MockUtil.serializableSpy(Employee.class, val);
```

Comment:
Could you schedule a talk about using Mockito on one of our Friday afternoons to educate us a bit? I know the basics and I need to scratch my head with spies. I'm sure that most other developers will be confused as well.